trackma/ui/qt/__init__.py

```diff
@@ -73,6 +73,10 @@ def main(force_qt4=False):
     app = QApplication(sys.argv)
     app.setApplicationName("trackma")
     app.setDesktopFileName("trackma")
+    if os.name == "nt":
+        import ctypes
+        myappid = u'trackma' + utils.VERSION
```

Comment:
Python 2 isn't supported by trackma, so you can drop the `u`.

```suggestion
        myappid = 'trackma' + utils.VERSION
```